DeepFried2/layers/SpatialMaxPooling.py

```diff
@@ -1,32 +1,74 @@
 import DeepFried2 as df
 from theano.tensor.signal.downsample import max_pool_2d
+import theano.tensor as _T
```

Comment:
Could you replace all `_T` by `df.T`? We used `_T` in the past, but I tried to change it everywhere.